.github/autolabeler.yml

```diff
@@ -0,0 +1 @@
+'autorelease': ['*/src/**/*.java']
```

Comment:
I've installed the relevant app already: https://github.com/apps/probot-autolabeler